changes/11666-add-nvd-resolved-version

```diff
@@ -0,0 +1 @@
+- (premium only) adds `resolved_in_version` to `/fleet/software` APIs pulled from NVD feed
```

Comment:
Let's add the change to the REST-API.md docs.

Reply:
we have a seperate docs PR: https://github.com/fleetdm/fleet/pull/13647